pilot/cmd/pilot-agent/main.go

```diff
@@ -48,6 +48,13 @@ import (
 	"istio.io/istio/pkg/spiffe"
 )
 
+const (
+	// TODO(quanlin): move udspath/tokenpath to env variable.
```

Comment:
Any reason not to make them environment variables now?

Reply:
This should be the default value.

Second is hardcoded by k8s - and hopefully we can stop using it.

Let's get this in to unblock sds install, we can decide if we need more knobs later.

Reply:
Can we get at least the prefix (/var/run) as an env variable, so we can test and run on VMs without root access ? We don't need all 3 - just the UDS path and a token path ( since we're debating if we should even support the unsecure one)